hydra-node/test/Hydra/Chain/Direct/ContractSpec.hs

```diff
@@ -170,6 +174,25 @@ prop_consistentHashPreSerializedCommits =
           & counterexample ("Serialized commits: " <> show serializedCommits)
           & counterexample ("To fanout txOuts: " <> show toFanoutTxOuts)
 
+prop_hashingCaresAboutOrderingOfTxOuts :: Property
+prop_hashingCaresAboutOrderingOfTxOuts =
+  forAllShrink genUTxOWithSimplifiedAddresses shrinkUTxO $ \(utxo :: UTxO) ->
+    (length utxo > 1)
+      ==> let plutusTxOuts =
```

Comment:
Suggestion: remove duplication here with the previous property?

Reply:
what do you mean?